app/persistence/fabric/CRUDService.ts

```diff
@@ -511,14 +536,20 @@ export class CRUDService {
 	 * @returns
 	 * @memberof CRUDService
 	 */
-	async getChannelsInfo(network_name) {
+	async getChannelsInfo(network_name, channel_genesis_hash) {
```

Comment:
As I see these are the changes to the existing function, please note that it shouldnt impact the application, if this function is called by any other functions.